cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -124,24 +132,52 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementHelper {
       }
     }
 
+  private def routeGetShardInfo(cluster: Cluster, shardRegionName: String) =
+    get {
+      extractExecutionContext { implicit executor =>
+        complete {
+          implicit val timeout = Timeout(5.seconds)
+          try {
+            ClusterSharding(cluster.system)
+              .shardRegion(shardRegionName)
+              .ask(ShardRegion.GetShardRegionStats)
+              .mapTo[ShardRegion.ShardRegionStats]
+              .map { shardRegionStats =>
+                ShardDetails(shardRegionStats.stats.map(s => ShardRegionInfo(s._1, s._2)).toSeq)
+              }
+          } catch {
+            case _: IllegalArgumentException =>
+              StatusCodes.NotFound → ClusterHttpManagementMessage(s"Shard Region $shardRegionName is not started")
+          }
+        }
+      }
+    }
+
   /**
    * Creates an instance of [[akka.cluster.http.management.ClusterHttpManagementRoutes]] to manage the specified
    * [[akka.cluster.Cluster]] instance. This version does not provide Basic Authentication. It uses
    * the default path "members".
    */
-  def apply(cluster: Cluster): Route = apply(cluster, "members")
+  def apply(cluster: Cluster): Route = apply(cluster, "")
 
   /**
    * Creates an instance of [[akka.cluster.http.management.ClusterHttpManagementRoutes]] to manage the specified
    * [[akka.cluster.Cluster]] instance. This version does not provide Basic Authentication. It uses
    * the specified path `pathPrefixName`.
    */
   def apply(cluster: Cluster, pathPrefixName: String): Route =
-    pathPrefix(pathPrefixName) {
-      pathEndOrSingleSlash {
-        routeGetMembers(cluster) ~ routePostMembers(cluster)
+    rawPathPrefix(pathPrefixName) {
+      pathPrefix("members") {
```

Comment:
it would. 
The old behavior was that the prefix would replace `members`. 
The behavior in this pr would change prefix to be a prefix of `members`. This is a more sustainable as we add more base REST resources to this library (in this case, `shards`). 

So here are our options:
1. make a breaking change to make `pathPrefix` a prefix of `/members, `/shards`, etc. (what this PR does as it stands). Perhaps, bump the version of this library to `0.4` to alert the breaking change. 

2. preserve old behavior: `pathPrefix` replaces `members` only, and create a new prefix parameter to prefix both `members` and `shards`, which we will use going forward. 

I like 1 because it's more flexible as we extend this library more and more. Since this library is pre 1.0 release, I feel like we have more flexibility to make these changes. That said @raboof let me know what you think